demo/item-userlists.ts

```diff
@@ -0,0 +1,244 @@
+/* eslint-disable no-param-reassign */
+import {
+  html,
+  css,
+  svg,
+  LitElement,
+  TemplateResult,
+  SVGTemplateResult,
+} from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import '../src/ia-icon-label';
+import { userlistDataInterface } from './item-userlists-model';
+
+interface userlistOptionInterface {
+  selectedHandler?: Function;
+  label: string | TemplateResult;
+  id: string;
+  isSelected?: boolean;
+}
+/**
+ * Component to display a list of userlists
+ * Used in ia-dropdown component
+ * to have different type and behavior of options
+ */
+@customElement('item-userlists')
+export class ItemUserlists extends LitElement {
+  /**
+   * Item identifier
+   */
+  @property({ type: String }) identifier = 'Flash';
+
+  /**
+   * List of item userlists
+   */
+  @property({ type: Array }) lists: userlistDataInterface[] = [];
+
+  // Set up Escape key listener after render
+  async firstUpdated(): Promise<void> {
+    await new Promise(resolve => setTimeout(resolve, 0));
+    document.addEventListener('keydown', (e: KeyboardEvent) => {
+      this.onKeydown(e);
+    });
```

Comment:
With the change to use an arrow function for `onKeyDown` (noted in another comment), this could be reduced to:

```suggestion
    document.addEventListener(this.onKeydown);
```

Reply:
Ok, thanks, done.